OpenRA.Mods.Cnc/Traits/Chronoshiftable.cs

```diff
@@ -205,13 +202,14 @@ public class ChronoshiftOriginInit : IActorInit<CPos>
 		readonly CPos value;
 
 		public ChronoshiftOriginInit(CPos init) { value = init; }
-		public CPos Value(World world) { return value; }
+		public CPos Value { get { return value; } }
 	}
 
-	public class ChronoshiftChronosphereInit : IActorInit<Actor>
+	public class ChronoshiftChronosphereInit : IActorInit
 	{
 		readonly Actor value;
 		public ChronoshiftChronosphereInit(Actor init) { value = init; }
-		public Actor Value(World world) { return value; }
+
+		public Lazy<Actor> Value(World world) { return new Lazy<Actor>(() => value); }
```

Comment:
This is a placeholder for the full implementation in the next PR that can initialize this with a map-placed actor (which may not exist until the end of the tick).

Reply:
Curious that you didn't remove the `World world` parameter too, but I suppose that the next PR will come in here.

Reply:
This is needed to query the `SpawnMapActors` trait.